.github/workflows/ci_tdd.yml

```
@@ -0,0 +1,29 @@
+---
+name: TDD
+on: 
+  pull_request:
+    branches:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+
+  validation:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+
+      - name: make the checkout in a sane way.
+        run: git clone https://github.com/${{ github.event.pull_request.user.login }}/galaxy_ng; cd galaxy_ng ; git checkout ${{ github.head_ref }}
```

```suggestion
        run: git clone https://github.com/${{ github.event.pull_request.user.login }}/galaxy_ng; cd galaxy_ng ; git checkout '${{ github.head_ref }}'
```

(branch names can contain parentheses)